osi_vehicle.proto

```diff
@@ -0,0 +1,926 @@
+syntax = "proto2";
+
+option optimize_for = SPEED;
+
+import "osi_version.proto";
+import "osi_common.proto";
+import "osi_object.proto";
+import "osi_occupant.proto";
+
+package osi3;
+
+message Traffic
```

Comment:
The idea was to have a "root message" with version and timestamp and the specific "VehicleClass". Now it has to be clarified what the exact purpose of this root message is. What model uses it as input/output? Consequently a OSMP example should be defined to clarifiy the usage.